lib/rules/no-unsupported-features/es-builtins.js

```diff
@@ -16,604 +16,604 @@ const trackMap = {
     globals: {
         // Core js builtins
         AggregateError: {
-            [READ]: { supported: "15.0.0" },
+            [READ]: { supported: ["15.0.0"] },
```

Comment:
There is only one supported version in this entire file, dispite that I converted them all to a range!